lib/numbers.md

```diff
@@ -5,20 +5,65 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+Integers (aka Fixnum) are whole numbers. 
+When used in mathematical operations they will always return whole numbers.
+
+    15/2 = 7
+    
+Floats are numbers with a decimal point. Floats great than -1 and less than 1 require a zero in front of the decimal point. 
+When used in mathematical operations they will always return floats.
+
+    15/2.0 = 7.5
```

Comment:
Curious that a math operation between an Integer and Float returns a Float. So Float takes precedence, in a sense.

(No revision necessary–just pointing that out.)